google/cloud/aiplatform/lro.py

```diff
@@ -15,6 +15,44 @@
 # limitations under the License.
 #
 
+import functools
 
-class LRO:
-    pass
+from google.api_core import operation as ga_operation
+from google.cloud.aiplatform_v1beta1.services.endpoint_service.transports.base import (
+    EndpointServiceTransport
+)
+from google.protobuf import empty_pb2 as empty
+
+
+class LRO(ga_operation):
+    """A handler for Operation futures.
+
+    Args:
+        operation (google.longrunning.operations_pb2.Operation): The
+            initial operation.
+        result_type (func:`type`): The protobuf type for the operation's
+            result.
+    """
+
+    def __init__(
+        self, operation, refresh=None, cancel=None, result_type=empty.Empty, **kwargs
+    ):
+        operations_client = EndpointServiceTransport().operations_client
```

Comment:
I don't think this works.  Need to create an operations_client in a similar way as the other resource noun classes.

Reply:
I think it does throw a NotImplementedError.  Should use the implementation from transports/grpc.py instead, *if* we are going to create the operations_client by borrowing from one of the services.  I think this is not the right way to do it, but we should just create an operations_client directly by providing the appropriate grpc channel.